docs/deprecated/distutils/apiref.rst

```diff
@@ -1755,7 +1756,7 @@ implementing.  This module should implement a class with the same name as the
 module (and the command).  So, for instance, to create the command
 ``peel_banana`` (so that users can run ``setup.py peel_banana``), you'd copy
 :file:`command_template` to :file:`distutils/command/peel_banana.py`, then edit
-it so that it's implementing the class :class:`peel_banana`, a subclass of
+it so that it's implementing the class ``peel_banana``, a subclass of
```

Comment:
Example hypothetical things don't really need to use the roles because using roles implies that there's someplace in the docs where the thing is supposed to be documented.